diff --git a/node_modules/js-autocomplete/auto-complete.js b/node_modules/js-autocomplete/auto-complete.js
index 2f5a88c..8d17fac 100644
--- a/node_modules/js-autocomplete/auto-complete.js
+++ b/node_modules/js-autocomplete/auto-complete.js
@@ -110,7 +110,7 @@ var autoComplete = (function(){
                     that.last_val = that.value;
                     that.sc.style.display = 'none';
                     setTimeout(function(){ that.sc.style.display = 'none'; }, 350); // hide suggestions on fast input
-                } else if (that !== document.activeElement) setTimeout(function(){ that.focus(); }, 20);
+                }// else if (that !== document.activeElement) setTimeout(function(){ that.focus(); }, 20);
             };
             addEvent(that, 'blur', that.blurHandler);
 
